### ISSUE TITLE
Would it be possible to share the implementation details for reproduction of examples in the paper?

### ISSUE THREAD

**Issue by hcwang95** [genforce/mganprior#7]
Thanks for the paper and the impressive results of the reconstruction, colorization etc.

Would you mind sharing more detailed hyper-parameters to help with the reproduction of the model? I am aware of there are some options (number of iterations, latent code initialization) not specified in the paper so I am wondering if it is possible to get their concrete values for different tasks. 

For your information, I've tried reconstruction of sample bedroom image using models pre-trained on bedroom, church and faces but I didn't get the almost perfect output as Fig 10 in the paper shows. Most of the time it is not sharp enough and sometimes there are unwanted textures visible from the pre-trained dataset.

Thanks a lot for the help! If not the full details of the experiment, any comment or suggestion on how to reproduce will be appreciated!

**Comment by JasonGUTU** [comment]
Hi,
Thanks for your interest. Do you run the inversion code with the default settings? And what is the used model? You can send some results to me that I can know more about your case. email: jinjingu@link.cuhk.edu.cn

**Comment by hcwang95** [comment]
Yeah I run the multi code inversion using default settings. The model is PGGAN pretrained on bedroom, church, and celebA. Thanks for your help and I will send the example images to your email!

**Comment by hcwang95** [comment]
Thanks @JasonGUTU for the suggestion. It turns out that the optimization matters to reproduce the result in Figure 10 in paper. After switching from Gradient Decent (default) to Adam with all other hyper-parameters remained the same, the reproduced outputs are almost the same as the ones in the paper.
